Service/SlidesInSlideDataCron.php

```diff
@@ -42,6 +42,10 @@ public function onCron(CronEvent $event)
         continue;
       }
 
+      if (count($subslides) < 1) {
```

Comment:
Does this still support the situation where I actually want a slide to be empty (eg. public service announcements)?

Reply:
The variable naming is not great here, but what it means is that if there are no content at all to put in even 1 slide, then skip.

It should be much better described, but it is the chance the event subscribers have to return an empty array if something went wrong.